### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by joshuaarmah**
#### Hi! What is your PR about?

Added a domain solarmoon.eth/vote.solarmoon.app

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy.

**Comment by joshuaarmah** [comment]
This is te snapshot URL
https://snapshot.org/#/vote.solarmoon.eth

Our space is under vote.solarmoon.app dns wise it is configured on our end

File: spaces/domains.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   "dao.growing.fi": "growingdao.eth",
   "dao.gnome.army": "gnometoken.eth",
   "vote.dsbswap.com": "dsbswap.eth",
+  "vote.solarmoon.app": "solarmoon.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:36]: Use your ENS here.
```suggestion
  "vote.solarmoon.app": "vote.solarmoon.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**